nbt/src/main/java/net/kyori/adventure/nbt/ListBinaryTag.java

```diff
@@ -366,6 +366,62 @@ default double getDouble(final @NonNegative int index, final double defaultValue
     return defaultValue;
   }
 
+  /**
+   * Gets a list.
+   *
+   * @param index the index
+   * @return the list, or an empty list
+   * @since 4.4.0
+   */
+  default @NonNull ListBinaryTag getList(final @NonNegative int index) {
+    return this.getList(index, null, ListBinaryTag.empty());
+  }
+
+  /**
+   * Gets a list.
+   *
+   * @param index the index
+   * @return the list, or an empty list
+   * @since 4.4.0
+   */
+  default @NonNull ListBinaryTag getList(final @NonNegative int index, final @Nullable BinaryTagType<?> elementType) {
+    return this.getList(index, elementType, ListBinaryTag.empty());
+  }
+
+  /**
+   * Gets a list.
+   *
+   * @param index the index
+   * @param defaultValue the default value
+   * @return the list, or {@code defaultValue}
+   * @since 4.4.0
+   */
+  default @NonNull ListBinaryTag getList(final @NonNegative int index, final @NonNull ListBinaryTag defaultValue) {
+    return this.getList(index, null, defaultValue);
+  }
+
+  /**
+   * Gets a list.
+   *
+   * <p>If {@code elementType} is non-{@code null} and the {@link ListBinaryTag} at index {@code index} does not match, {@code defaultValue} will be returned.</p>
+   *
+   * @param index the index
+   * @param elementType the element type of the list
```

Comment:
since there are two lists in context here, i'd like to see the JD be super explicit about which list is relevant for any operation or parameter.